cookiecutter.json

```diff
@@ -1,6 +1,20 @@
 {
     "project_name": "Template",
     "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
-    "libmode": "git.submodule",
-    "include_library": ["SeismicPro", "SeismiQB", "PetroFlow"]
+    "include_library": ["no", "SeismicPro", "seismiqb", "petroflow", "batchflow"],
+    "git_config_user_name": "do_not_set",
+    "git_config_user_email": "do_not_set",
+    "enable_slack_notification": ["no", "yes"],
+    "init_DVC": ["yes", "no"],
```

Comment:
Two consequent parameters have different order of options `["no", "yes"]`

Reply:
Yes, if a list of options is provided, the first one becomes the default, so I put on the first place what seemed reasonable to me. Any suggestions on most convenient default values are welcome!